docs/interact/index.mdx

```diff
@@ -0,0 +1,41 @@
+# Interact with Feldera
+
+There are four different ways to interact with Feldera. All methods allow users to define and run pipelines,
+monitor their performance and interact with materialized tables, views and change streams.
+
+```mdx-code-block
+import Link from '@docusaurus/Link';
+
+<div className="card">
+  <div className="card__body">
+    <h3>
+      <Link to="/concepts">Web Console</Link>
+    </h3>
+    <p>The Web Console is the most convenient starting point to using Feldera. Simply aim your web browser at a running Feldera instance (and authenticate if required). This is typically `localhost:8080` if you are using our Docker instance or `try.feldera.com` if you are using our public cloud sandbox.</p>
+  </div>
+</div>
+<div className="card">
+  <div className="card__body">
+    <h3>
+      <Link to="/reference/cli">Commandline Interface (`fda`)</Link>
+    </h3>
+    <p>`fda` allows users to interact with a Feldera instances from the CLI. It also provides a shell that makes it convenient to issue ad-hoc queries and start/stop pipelines.</p>
+  </div>
+</div>
+<div className="card">
+  <div className="card__body">
+    <h3>
+      <Link to="/python/index.html">Python SDK</Link>
+    </h3>
+    <p>The Python SDK provides a convenient wrapper around our REST API. It allows you to programmatically define pipelines.</p>
+  </div>
+</div>
+<div className="card">
+  <div className="card__body">
+    <h3>
+      <Link to="/api">REST API</Link>
+    </h3>
+    <p>All the above methods interact with Feldera using the REST API. This is intended for users who would like to build their own clients to Feldera. `Pipeline` is the most important API type. </p>
```

Comment:
This should say something about automating Feldera operations (platform engineers probably don't think of themselves as building clients)